src/pages/Detail/components/DetailArticle/DetailArticle.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export default function DetailArticle() {
+  return (
+    <UploadWrapper>
+      <article>
+        <div className="imageWrapper">
```

Comment:
```suggestion
        <ImageWrapper>
```
어차피 클래스명으로 이름지을거라면 스타일드 컴포넌트로 만들어주세요
다른 부분도 수정해주세요

Reply:
넵 해당사항 확인했고 작성한 코드들 수정할게요!